src/styles/main.scss

```
@@ -1 +1,84 @@
-// put code here
+.body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+  align-items: center;
+}
+
+
+.stopwatch {
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__center {
+    box-sizing: border-box;
+    position: relative;
+    height: 5vmin;
+    width: 5vmin;
+    border-radius: 100%;
+    background-color: #f6a603;
+  }
+
+  &__seconds-hand {
+    position: absolute;
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+    bottom: 50%;
+    left: 50%;
+    animation: rotate 60s linear infinite running;
+    transform-origin: bottom center;
+    z-index: -1;
+  }
+
+  &__minutes-hand {
+    position: absolute;
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    bottom: 50%;
+    left: 50%;
+    animation: rotate 3600s steps(60) infinite running;
+    transform-origin: bottom center;
+    z-index: -2;
+  }
+
+  &--speed-up .stopwatch__seconds-hand {
+    animation: rotate 10s linear infinite running;
+  }
+
+  &--speed-up .stopwatch__minutes-hand {
+    animation: rotate 600s steps(60) infinite running;
+  }
+}
+
+@keyframes rotate {
+  0% {
+    transform: translate(-50%) rotate(0deg);
+  }
+
+  25% {
+    transform: translate(-50%) rotate(90deg);
+  }
+
+  50% {
+    transform: translate(-50%) rotate(180deg);
+  }
+
+  75% {
+    transform: translate(-50%) rotate(270deg);
+  }
+
+  100% {
+    transform: translate(-50%) rotate(360deg);
+  }
+}
```

1. move translate(-50%) outside the animation (to your class selectors)
2. no needs to such a big amount of keyframes
```suggestion
@keyframes rotate {
  0% {
    transform: rotate(0deg);
  }

  100% {
    transform: rotate(360deg);
  }
}
```

---

I had to leave 25% because tests wouldn't pass otherwise